cmd/deploy/remote.go

```diff
@@ -74,6 +78,8 @@ COPY . /okteto/src
 WORKDIR /okteto/src
 
 ENV OKTETO_INVALIDATE_CACHE {{ .RandomInt }}
+ARG OKTETO_TLS_CERT_BASE64
```

Comment:
We should also do this fro `cmd/destroy/remote.go`

Reply:
done